bitcoin/src/blockdata/transaction.rs

```diff
@@ -61,7 +61,7 @@ crate::serde_utils::serde_struct_human_string_impl!(OutPoint, "an OutPoint", txi
 
 impl OutPoint {
     /// The number of bytes that an outpoint contributes to the size of a transaction.
-    const SIZE: usize = 32 + 4; // The serialized lengths of txid and vout.
+    pub const SIZE: usize = 32 + 4; // The serialized lengths of txid and vout.
```

Comment:
If we're making it public we should rename it to `SERIALIZED_SIZE`. I couldn't guess what it is until I've read the doc.